### ISSUE TITLE
数据集

### ISSUE THREAD

**Issue by fzuqxt** [HITRainer/EAVSR#2]
作者您好，请问数据集是否损坏，下载了有些图片不能解压

**Comment by 1180300419** [comment]
您好，现在问题解决了嘛？
数据集应该是没问题的，重新下载损坏的压缩包试试呢？

**Comment by fzuqxt** [comment]
谢谢，已经解决了，还有个问题，请问下LR输入网络和HR尺寸一样吗，需要先把LR下采样吗



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Xiaohui Liu ***@***.***&gt;
发送时间: 2023年11月2日 11:47
收件人: HITRainer/EAVSR ***@***.***&gt;
抄送: fzuqxt ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [HITRainer/EAVSR] 数据集 (Issue #2)





 
您好，现在问题解决了嘛？
 数据集应该是没问题的，重新下载损坏的压缩包试试呢？

**Comment by 1180300419** [comment]
在dataset的代码里面，有对图片进行crop的操作，输入网络的LR和HR的大小比例是1：scale。
不需要另外进行crop操作。
具体代码可见：https://github.com/HITRainer/EAVSR/blob/main/data/mvsr4x_dataset.py#L166

**Comment by HITRainer** [comment]
@fzuqxt That's right, you need to down-sample the LR sequences since we supply the same size of LR/HR data pairs. 
Actually, if you use our code to pre-process the dataset, you don't need to do downsampling operation yourself, you can refer to [DATA PRE-PROCESSING](https://github.com/HITRainer/EAVSR/blob/main/data/mvsr4x_dataset.py#L199)
